README.md

```diff
@@ -18,6 +18,8 @@ This repository contains [Jetpack Compose][jetpack-compose] components for the [
 
 ## Installation
 
+You no longer need to specify the Maps SDK for Android or its Utility Library as separate dependencies, since `android-maps-compose` and `android-maps-compose-utils` pull in the appropriate versions of these respectively.
```

Comment:
oops, the dependencies are just `maps-compose` and `maps-compose-utils`

Also down below in the groovy snippet, we should delete L27-29